### ISSUE TITLE
Install Fails With "Command not found" ... But Other Packages Install Fine?

### ISSUE THREAD

**Issue by machineghost** [cameronhunter/local-ssl-proxy#105]
As described in this Stack Overflow question (https://stackoverflow.com/questions/74669095/npm-install-fails-with-command-not-found-only-with-one-package), I'm having a problem installing this library ... and *only* this library.

When I try to, I get a "Command not found" error, which should translate into "you don't have `npm` installed idiot" ... but I can install other libraries (including ones with hyphens in their names) just fine.

It really looks like a problem with my system, but since it seems exclusive to this library I thought I'd check and see if this was somehow a (very strange) known issue.

**Comment by cameronhunter** [comment]
I can't imagine what's happening here, so I will close the issue. Please investigate the issue and report back if you find anything.

**Comment by sefrijn** [comment]
I have the same issue if I install the latest version (2.0.2)
Using MacOS 13.0.1 with Node 18.14.2

Steps to reproduce:
```
npm install -g local-ssl-proxy
local-ssl-proxy --version
zsh: command not found: local-ssl-proxy
```

If I install version 1.3.0 it works just fine, so I will stick to that.
All my other packages (global and local) work just fine.

**Comment by cameronhunter** [comment]
I had made a mistake with the `bin` path in `package.json`. I've published `v2.0.4` which resolves the issue. You can verify with `npx local-ssl-proxy@2.0.4`
